exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = (htmlLevel > 5);
+let cssLevelAbove5 = (cssLevel > 5);
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-let cssAndHtmlAbove5;
-let cssOrHtmlAbove5;
+let cssAndHtmlAbove5 = (cssLevel>5 && htmlLevel>5);
```

Comment:
Could you think if you could only variables defined above?

e.g. let cssAndHtmlAbove5 = cssLevelAbove5 && htmlLevelAbove5

Would you apply this to the next line too?